src/olympia/yara/migrations/0004_delete_yararesult.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 2.2.6 on 2019-10-18 07:52
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('yara', '0003_auto_20191010_1446'),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
+            name='YaraResult',
```

Comment:
We probably don't want to execute this migration in the same week as the one that replaces `YaraResult` by `ScannerResult` to keep backwards-compatibility during deploy... 

... Although, maybe in this case we could cheat and disable the switch before to be sure there are no errors, and re-enable it after the deploy?

Reply:
I'd vote for disabling the switch or put the site in read only mode because otherwise we'll have to copy the data from `yara_results` to `scanners_results` twice and/or force admins to look at 2 django admins to see the results.